File: Code-Chef/Reverse-The-Number/reverse_the_number.cpp
/*  Problem Statement: https://www.codechef.com/problems/FLOW007
    Author: striker
*/

#include <bits/stdc++.h>

static unsigned int compute_reverse_number(unsigned int);

int main(void) {
    int test;
    std :: cin >> test;
    while(test--) {
        unsigned int n;
        std :: cin >> n;
        std :: cout << compute_reverse_number(n) << std :: endl;
    }
    return 0;
}

static unsigned int compute_reverse_number(unsigned int n) {
    unsigned int reverse_number = 0;
    while(n) {
        reverse_number = (10 * reverse_number) + (n % 10);
        n /= 10;
    }
    return reverse_number;
}

File: Code-Chef/Reverse-The-Number/reverse_the_number.py
'''
Problem Statement: https://www.codechef.com/problems/FLOW007
Author: striker
'''

def compute_reverse_number(number):
    return int(number[: : -1])

def main():
    for test in range(int(input().strip())):
        print("{0}".format(compute_reverse_number(input().strip())))

if __name__ == "__main__":
    main()
